web/pages/embed/analytics.tsx

```diff
@@ -5,7 +5,7 @@ import Analytics, {
   CustomAnalytics,
```

Comment:
Move this file to pages/embed/stats.tsx? Would break existing embeds I guess

Reply:
Yeah I didn't do this because I didn't want to break existing embeds. Not sure if a redirect would solve it, but I asusme it's not as much of a problem.

One reason to do this would be if existing embeds are getting flagged by adblock too, but I haven't heard any reports of this.

Reply:
Interesting. I wonder what the relevant adblock rules look like.